Sources/SwerverCore/Server/ClientRequestHandler.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import Socket
+
+public class ClientRequestHandler {
+    
+    private let httpRequestParser = HttpRequestParser()
+    private let httpRouteProcessor = Router()
+    
+    public init() {}
+    
+    public func handle(client: Socket) throws {
+        let httpRequest = try parseRequest(clientSocket: client)
+        try client.write(from: httpRouteProcessor.process(request: httpRequest).responseDataToString())
+    }
+    
+    private func parseRequest(clientSocket: Socket) throws -> HttpRequest {
+        var readData: Data? = Data(capacity: Swerver.bufferSize)
+        let _ = try clientSocket.read(into: &readData!)
```

Comment:
Do you need `let _`?

Reply:
I put let _ there just to suppress the warning from xcode of the value being unused.